backend/src/database/migrations/20230331185335_create_db_lock_table.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex) {
+  return knex.schema.withSchema("data").createTable("db_lock", function (table) {
+    table.increments().primary(); //id column
+    table.string("locked_table");
+    table.string("locked_by");
+    table.integer("locked_row_id").unsigned();
+    table.date("locked_date");
+  });
+};
+
+exports.down = function (knex) {
+  return knex.schema.dropTable("users");
```

Comment:
This should be dropping "data.db_lock" or however it's referenced.

Reply:
Thank goodness you caught this lol!!! my bad